apps/b2b/components/Auth/LoginForm.tsx

```diff
@@ -0,0 +1,60 @@
+import { yupResolver } from '@hookform/resolvers/yup'
+import ErrorMessage from 'components/ErrorMessage'
+import Input from 'components/Input'
+
+import { signIn } from 'next-auth/react'
+import { useForm } from 'react-hook-form'
+import { Button } from 'ui'
```

Comment:
let's import from ui/components/Button


https://www.telerik.com/blogs/tree-shaking-basics-for-react-applications


we should probably remove that index file in ui too.

Reply:
will check this out! also is there a way for us to use absolute import instead of relative? for tsconfig? Incase we needed to move components around